src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -22,9 +22,8 @@
 #include <ripple/app/ledger/InboundLedgers.h>
 #include <ripple/app/ledger/LedgerMaster.h>
 #include <ripple/app/misc/LoadFeeTrack.h>
-#include <ripple/core/ThreadEntry.h>
 #include <ripple/protocol/JsonFields.h>
-#include <ripple/beast/core/Thread.h>
+#include <ripple/beast/core/CurrentThreadName.h>
```

Comment:
I dug into this a bit further.  On MacOS it looks like the files included by `beast/core/core.unity.cpp` (which at the moment includes `CurrentThreadName`) get special treatment so they can interoperate with Objective C.  It also looks like beast/core is the only place that gets this special treatment -- that's the only place I see `*.mm` files for Objective C.

So my inclination is to keep `CurrentThreadName` in `ripple/beast/core`, along with the other files that solve awkward OS integration problems.  Let me know what you think.  Thanks.